terraso_backend/apps/core/models/users.py

```diff
@@ -71,6 +71,11 @@ class User(SafeDeleteModel, AbstractUser):
     email = models.EmailField()
     profile_image = models.URLField(blank=True, default="")
 
+    # Alternatives to is_test_user?:
+    # 1. Add a fixed prefix to the email address
+    # 2. Create a separate table to store what users are test users
+    is_test_user = models.BooleanField(blank=True, default=False)
```

Comment:
I don't think it makes sense to add a flag to the model. Having another table might be okay. Please do further research in this area.

Reply:
@paulschreiber There is not a lot of specific information about how to implement this, there is a lot of mentions related to isolating test data, so knowing what are the test users and even potentially identify what have they done in the system is important. With that in mind I do agree that the core app should not know details about testing, so I think a separate table makes sense, should this be a separate app intended for e2e tests?

Reply:
This is the most relevant conversation I found related to the topic, but no mention about implementation details: https://stackoverflow.com/questions/55093017/should-e2e-tests-persist-data-in-real-databases